atlas_direction_vectors/algorithms/direction_vectors_from_center.py

```diff
@@ -0,0 +1,48 @@
+"""Generic algorithm for the computation of direction vectors.
+
+This algorithm applies to any brain region. A point is selected
+(by default the center of the brain) and direction vectors are
+establieshed as pointing away from said point.
+
+This algorithm is intended to generate placeholder directions.
+"""
+import numpy as np
+import numpy.typing as npt
```

Comment:
we normally use `npt` for `numpy.testing`; I think we shouldn't create the `as` alias here.